dao/authentication_dao.go

```diff
@@ -398,6 +434,49 @@ func authFromVault(secret *api.Secret) *m.Authentication {
 		auth.SourceID = id
 	}
 
+	// Request the JWT token if the authentication is a "marketplace" authentication
+	if auth.Name == "marketplace" {
```

Comment:
Super small nit - can we add a log message here saying we're going to include the jwt token in the response?

Reply:
Done. Before resolving this one: I make the assumption that the authentication's name will be stored as `"marketplace"`, but maybe I'm making a too big of an assumption. Can we somehow force in [`sources-api/db/seeds/source-types.yml`](https://github.com/RedHatInsights/sources-api/blob/master/db/seeds/source_types.yml#L267-L285) which name will the auth take when it gets persisted to the db?

Reply:
@lindgrenj6 I'm assuming that the `auth.Name` will be "marketplace", but as I say, it's an assumption. Can we force a "marketplace" auth to have the name "marketplace" somehow? Thanks in advance!